mslib/mscolab/server.py

```diff
@@ -242,16 +242,19 @@ def hello():
             auth.login_required()
             return json.dumps({
                 'message': "Mscolab server",
-                'USE_SAML2': mscolab_settings.USE_SAML2
+                'USE_SAML2': mscolab_settings.USE_SAML2,
+                'direct_login': mscolab_settings.DIRECT_LOGIN
```

Comment:
We should decide if keys in this json should be uppercase or lowercase, not mix them. I'd prefer lowercase.

Reply:
Yeah.. Thank you for figguring out that point. I believe that the `USE_SAML2` configuration pertains to a previous implementation and is not directly relevant to the current issue at hand. Consequently, it may be prudent to consider opening a separate issue, as the matter at hand is distinct and not associated with the present concern. This step is suggested to enhance clarity and maintain a focused approach to problem resolution.

Reply:
If we decide on lowercase then yes, this does not block this PR and should be an issue instead. I just wanted to bring it up, in case someone disagreed with me preferring lowercase.